src/redprl/refiner.fun

```diff
@@ -17,7 +17,7 @@ struct
 
   structure Hyp =
   struct
-    fun Project z alpha jdg =
+    fun Project z _ jdg =
```

Comment:
I'm also not a big fan of `_(*alpha*)`, as it seems very bureaucratic and is kind of confusing to look at; moreover, it makes the code more sensitive to small changes. Having to comment out a variable binding non-locally if I remove a line that is the only use of some variable is kind of odious to me.